app/experimenter/nimbus-ui/src/components/LinkNav/index.tsx

```diff
@@ -17,6 +17,7 @@ type LinkNavProps = {
   className?: string;
   textColor?: string;
   title?: string;
+  onClick?: any;
```

Comment:
I'd like to avoid `any` where possible - I believe this could be `() => void`

Reply:
Sploops! Forgot I left that in there 👍